lib/fog/parsers/terremark/task.rb

```diff
@@ -9,12 +9,12 @@ def reset
         def start_element(name, attributes)
           super
           case name
-            when 'Owner', 'Result', 'Link', 'Error'
-              data = extract_attributes(attributes)
-              @response[name] = data
-            when 'Task'
-              task = extract_attributes(attributes)
-              @response.merge!(task.reject {|key,value| !['endTime', 'href', 'startTime', 'status', 'type'].include?(key)})
+          when "Owner", "Result", "Link", "Error"
+            data = extract_attributes(attributes)
+            @response[name] = data
+          when "Task"
+            task = extract_attributes(attributes)
+            @response.merge!(task.reject {|key,value| !["endTime", "href", "startTime", "status", "type"].include?(key)})
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.<br>Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.